File: 1_mann_whitney_u_test.r
# Read the CSV data into a DataFrame
data <- read.csv("your_data.csv")

# Perform the Mann-Whitney U test
result <- wilcox.test(data$Population, data$AverageIncome)

# Print the test statistic and p-value
cat("Mann-Whitney U test:\n")
cat("Test Statistic:", result$statistic, "\n")
cat("p-value:", result$p.value, "\n")

File: 1_mann_whitney_u_test.py
import pandas as pd
from scipy.stats import mannwhitneyu

# Read the CSV data into a DataFrame
data = pd.read_csv('your_data.csv')

# Perform the Mann-Whitney U test
result = mannwhitneyu(data['Population'], data['AverageIncome'])

# Print the test statistic and p-value
print("Mann-Whitney U test:")
print("Test Statistic:", result.statistic)
print("p-value:", result.pvalue)
